crates/bevy_sprite/src/render/sprite.vert

```diff
@@ -17,8 +17,16 @@ layout(set = 2, binding = 1) uniform Sprite {
     vec2 Sprite_size;
 };
 
+layout(set = 1, binding = 1) uniform ColorMaterial_flip_horz {
```

Comment:
This is a valuable feature, but creating two new bindings for it seems a bit too much. I understand that the current "data driven" bindings encourage this design, but I don't want to go too far down that path. Ideally we either:
1. make it easier to consolidate these bindings into one
2. make these "ShaderDefs" instead (but that also has the problem of forcing more state changes / reducing batchability).

Reply:
Agreed. I want to make an issue for the bindings as it seems like this is something the average user is going to run into when creating a custom shader.